lib/presentation/screen/journal_screens/journal_create_screen.dart

```diff
@@ -96,11 +95,11 @@ class _JournalCreateScreenState extends State<JournalCreateScreen> {
             ),
             floatingActionButton: SecondaryFloatingActionButton(
               onTap: () async {
```

Comment:
Remove async if we are not using 'await'

Reply:
ok